src/app/pages/model-3d/model-3d-list/model-3d-list.page.html

```diff
@@ -0,0 +1,40 @@
+<app-general-container-component>
+  <div class="d-flex justify-content-between align-items-center flex-wrap">
+    <h1>Modelo de tu mascota en 3D</h1>
+    <button
+      matButton="elevated"
+      class="mb-3"
+      (click)="navigateToCreateModel3D()"
+    >
+      <mat-icon>add</mat-icon>
+      Crear modelo 3D
+    </button>
+  </div>
+</app-general-container-component>
+<form [formGroup]="form">
+  <div class="row">
+    <div class="col-lg-6 mb-3">
+      <mat-form-field appearance="outline" class="w-100">
+        <mat-label>Animal</mat-label>
+        <mat-select formControlName="animalId" required>
+          <mat-option *ngFor="let animal of animals" [value]="animal.id">
```

Comment:
cambie esto por la estructura de control moderna de angular 

@for 


```suggestion
    @for (animal of animals) {
       <mat-option  [value]="animal.id">
    }        
```

Nota importante. Veo que usas [value] y le metes el id del animal. Tambien puedes meter directamente el objeto animal... por si acaso.. a veces es util tener todo el objeto directamente.

seria algo como 

```typescript
    @for (animal of animals) {
       <mat-option  [value]="animal">
    }       
```